api/src/database/migrations/run.ts

```diff
@@ -24,7 +24,9 @@ export default async function run(database: Knex, direction: 'up' | 'down' | 'la
 	const migrations = [
 		...migrationFiles.map((path) => parseFilePath(path)),
 		...customMigrationFiles.map((path) => parseFilePath(path, true)),
-	].sort((a, b) => (a.version > b.version ? 1 : -1));
+	]
+		.sort((a, b) => (a.version > b.version ? 1 : -1))
+		.filter((m) => m.file.endsWith('.js'));
```

Comment:
We should be able to move this to line 18 since there's existing filtering logic there, with customMigrations' filtering in line 20.